index.ts

```diff
@@ -0,0 +1,2112 @@
+// Generated by @wagmi/cli@1.0.0 on 5/10/2023 at 4:08:29 PM
```

Comment:
Could these be in separate files and re-exported in an index.ts file?

Reply:
not sure if its currently possible; we'd likely have to do a diff wagmi config for each abi.  lets leave this to a v2.